src/process_engine_service.ts

```
@@ -126,7 +133,25 @@ export class ProcessEngineService implements IProcessEngineService {
     return this._internalContext;
   }
 
+  private _initializeDefaultErrorDeserializer(): void {
+    const defaultDeserializer: IErrorDeserializer = (serializedError: string): Error => {
+
+      if (typeof serializedError !== 'string') {
+        return serializedError;
+      }
+
+      try {
+        return ProcessEngineErrors.BaseError.deserialize(serializedError);
+      } catch (error) {
+        return undefined;
```

Why hide errors that can't be deserialized?

---

The consequence would be, that in a `catch`-block you get any unknown string as the error. I think that is more confusing than it helps. What do you think about throwing a new error instead, with a message that explains, that the problem was a missing error deserialization?

---

@5ebastianMeier I like that approach with the new Error